telco-hub/configuration/example-overlays-config/logging/cluster-log-forwarder-patch.yaml

```diff
@@ -0,0 +1,13 @@
+---
+# patching ClusterLogForwarder for hub-specific configuration
+- op: replace
+  path: /spec/outputs/0/kafka/url
+  value: tcp://jumphost.inbound.bos2.lab:9092/{{ fromClusterClaim "name" }}
```

Comment:
This configuration does not get run through the policy engine so I don't think templating will work in this case.

Reply:
Agree, thanks for pointing this out.